packages/shared/src/components/modals/report/ReportComentModal.spec.tsx

```diff
@@ -125,14 +125,16 @@ it('submit the report with text', async () => {
     },
   });
 
-  otherRadio.click();
-  await expect(otherRadio).toBeChecked();
+  await act(async () => {
+    otherRadio.click();
+    await expect(otherRadio).toBeChecked();
 
-  const input = screen.getByTestId('report_comment') as HTMLTextAreaElement;
-  fireEvent.change(input, { target: { value: 'test note' } });
-  input.dispatchEvent(new Event('input', { bubbles: true }));
+    const input = screen.getByTestId('report_comment') as HTMLTextAreaElement;
+    fireEvent.change(input, { target: { value: 'test note' } });
+    input.dispatchEvent(new Event('input', { bubbles: true }));
+    expect(input.value).toBe('test note');
+  });
```

Comment:
Without the `act`, I get this error below.

![Screenshot 2024-06-29 at 11 13 51 AM](https://github.com/dailydotdev/apps/assets/13744167/f65eb08a-35c4-4a70-a73c-6f07ffb97743)